pydocx/fixtures/export_from_googledocs.html

```diff
@@ -0,0 +1 @@
+<html><head><meta charset="utf-8" /><style>.pydocx-caps {text-transform:uppercase}.pydocx-center {text-align:center}.pydocx-comment {color:blue}.pydocx-delete {color:red;text-decoration:line-through}.pydocx-hidden {visibility:hidden}.pydocx-insert {color:green}.pydocx-left {text-align:left}.pydocx-right {text-align:right}.pydocx-small-caps {font-variant:small-caps}.pydocx-strike {text-decoration:line-through}.pydocx-tab {display:inline-block;width:4em}.pydocx-underline {text-decoration:underline}body {margin:0px auto;width:51.00em}</style></head><body><p>Beauty will save the world.</p></body></html>
```

Comment:
This should just be `<p>Beauty will save the world.</p>`, see the other fixture html files for examples.